samples/modules/lvgl/multi_display/Kconfig

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2023 Fabian Blatz <fabianblatz@gmail.com>
+# Copyright (c) 2025 Abderrahmane JARMOUNI
+# SPDX-License-Identifier: Apache-2.0
+
+choice LV_Z_DEMO_FIRST_DISP
+	prompt "LVGL demo to build for 1st display"
+	default LV_Z_DEMO_BENCHMARK_FIRST_DISP
+	help
+	  Demo application to build for 1st display.
+
+config LV_Z_DEMO_BENCHMARK_FIRST_DISP
+	bool "LVGL benchmark demo"
+	select LV_USE_DEMO_BENCHMARK
+	select LV_USE_SYSMON
+	select LV_USE_PERF_MONITOR
+	help
+	  Build benchmarking demo application.
+
+config LV_Z_DEMO_STRESS_FIRST_DISP
+	bool "LVGL stress demo"
+	select LV_USE_DEMO_STRESS
+	select LV_USE_SYSMON
+	select LV_USE_PERF_MONITOR
+	help
+	  Build stress testing demo application.
+
+config LV_Z_DEMO_WIDGETS_FIRST_DISP
+	bool "LVGL widgets demo"
+	select LV_USE_DEMO_WIDGETS
+	help
+	  Build widgets demo application.
+
+config LV_Z_DEMO_FLEX_LAYOUT_FIRST_DISP
+	bool "LVGL flex layout demo"
+	select LV_USE_DEMO_FLEX_LAYOUT
+	help
+	  Build flex layout demo application.
+
+config LV_Z_DEMO_SCROLL_FIRST_DISP
```

Comment:
LVGL upstream has recently moved some of the newer demos to a new repository.
https://github.com/lvgl/lvgl/pull/8022

Since I do not see having a fork of that one as well or maintaining the sample in tree, as pseudo mirror, this sample will probably be removed in the next update of our LVGL fork. I mean it works but I guess it would make things slimer if we did not add it just to remove it a few weeks in the future.